src/pages/WishList/component/Wishlist-card.jsx

```diff
@@ -0,0 +1,33 @@
+import "./Wishlist-card.css"
+const WishlistCard = ({
+    cardImage,
```

Comment:
Here you can use prop as parameter as destructuring is making code bulky.

Reply:
> Here you can use prop as parameter as destructuring is making code bulky.

Yes thanks will use props.